lib/config-schema.ts

```diff
@@ -0,0 +1,71 @@
+import { z } from "zod";
+
+
+// Used for defaults, and a partial used for org/account overrides
+const Settings = z.object({
+    region: z.string().optional(),
+    timezone: z.string().default('utc'),
+    timezoneTag: z.string().default('Timezone'),
+    organizationRoleName: z.string(),
+    revolverRoleName: z.string(),
+    saveResources: z.string(),
+});
+
+export default z.object({
+    defaults: z.object({
+        settings: Settings,
+        drivers: z.array(
+            z.object({
+                name: z.string(),
+                active: z.boolean().default(true),
+                pretend: z.boolean().default(false)
+            }),
+        ).default([]),
+        plugins: z.object({
+            powercycle: z.object({
+                active: z.boolean(),
+                configs: z.tuple([
+                    z.object({
+                        tagging: z.string().default('strict'),
+                        availabilityTag: z.string().default('Schedule')
+                    }),
+                ]),
+            }).optional(),
+            validateTags: z.object({
+                active: z.boolean(),
+                configs: z.array(
+                    z.object({
+                        tag: z.string(),
+                        tagMissing: z.array(
+                            z.union([z.string(), z.object({ setDefault: z.string() })])
+                        ),
+                        onlyResourceTypes: z.array(z.string()),
+                        tagNotMatch: z.array(z.any()),
+                    }),
+                ),
+            }).optional(),
+        }),
```

Comment:
Default empty plugins?
```suggestion
        }).default([]),
```

Reply:
hmm, it's an object at the moment